app/controllers/application_controller.rb

```diff
@@ -860,19 +860,12 @@ def get_reports_menu(group = current_group, tree_type = "reports", mode = "menu"
     folders = []
     user = current_user
     @sb[:grp_title] = reports_group_title
-    @data = []
+    data = []
```

Comment:
Was `@data` not used elsewhere?  If so, was it just a local variable masquerading an instance variable.